src/utils/data.js

```diff
@@ -1,4 +1,5 @@
 import { isString, escapeHtml } from "./utilities";
+import jsonpath from 'jsonpath';
```

Comment:
do we really need extra library for that? is there an alternative maybe?
it's 403 kb unpacked and kinda heavy for one place usage

Reply:
I've changed to use lodash.get, I would imagine the size would be smaller